src/ui/widgets/widgetActions.ts

```diff
@@ -98,6 +102,12 @@ export const getActionDescription = (action: WidgetAction): string => {
       } else {
         return `Close tab ${action.dynamicInfo.name}`;
       }
+    case OPEN_DISPLAY:
+      if (action.dynamicInfo.description) {
+        return action.dynamicInfo.description;
+      } else {
+        return `Open display ${action.dynamicInfo.name}`;
+      }
```

Comment:
Currently have made a copy of `OPEN_TAB` (which I'm sure whether it serves the same purpose or not, and called it `OPEN_DISPLAY` instead. Is there an existing DynamicContent property that has the functionality these new pages want? I couldn't find any documentation on the existing properties so wasn't sure.

Reply:
I'll have a look. Do we not want to use `OPEN_TAB`?

Reply:
I'm not sure what we want to use, I didn't know whether one of these actions was designed specifically with this purpose in mind or whether a new one is needed.

Reply:
No, I think we want `OPEN_TAB` here.

Reply:
In this case can we rename `OPEN_TAB` to `OPEN_DISPLAY` as "OPEN_DISPLAY" is how it comes in on the props from the .opi. Also line 191 of opiParser would be more consistent with the rest of the if else statements.

Reply:
Just looked at the background code, and it would be a pain to change all instances of tab to display, so perhaps could just be changed in widgetActions, as widgetActions makes the calls to the fileContext.

Reply:
I don't particularly feel the need to change this one as I think it makes sense for our app. `OPEN_PAGE` will open the file in any `DynamicPage` with the right location and `OPEN_TAB` will open the file in any `DynamicTabs` with the right location.

Reply:
That's fine, will just have to map the `OPEN_DISPLAY` to `OPEN_TAB` then.